CHANGELOG.md

```diff
@@ -217,7 +218,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Changed
 
-- The parameter `--path` of `tuist graph` now specifies where the manifest is. To specify the output directory of the graph, use `--output-path` [#2434]((https://github.com/tuist/tuist/pull/2434) by [@adellibovi](https://github.com/adellibovi).
+- The parameter `--path` of `tuist graph` now specifies where the manifest is. To specify the output directory of the graph, use `--build-output-path` [#2434]((https://github.com/tuist/tuist/pull/2434) by [@adellibovi](https://github.com/adellibovi).
```

Comment:
I believe this is not correct?

Reply:
oops, i replaced `--output-path` so it did this too lol, going to undo